src/app/auth/reset-password/reset-password.page.ts

```
@@ -1,43 +1,71 @@
-  sendResetLink(email: string) {
+  sendResetLink(email: string): void {
     this.isLoading = true;
+    this.resetEmail = email;
+
+    if (this.currentPageState === PageState.success) {
+      this.isEmailSentOnce = true;
+    }
 
     this.routerAuthService
       .sendResetPassword(email)
       .pipe(tap(() => (this.isLoading = false)))
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)